src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -372,7 +372,7 @@ public function send()
 
         if (function_exists('fastcgi_finish_request')) {
             fastcgi_finish_request();
-        } elseif ('cli' !== PHP_SAPI) {
+        } elseif (!in_array(PHP_SAPI, array('cli', 'phpdbg'))) {
```

Comment:
Could you update your code with `} elseif (!\in_array(PHP_SAPI, array('cli', 'phpdbg'), true)) {`

Reply:
Done.